client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -718,4 +718,4 @@ public List<PendingEvent> getUnackedEventsOnSeal() {
     public long getLastObservedWriteOffset() {
         return state.getLastSegmentLength();
     }
-}
+}
```

Comment:
Please ignore this, tried multiple time, I am not getting any diff with master in this file.